tools/mason/MasonHelp.chpl

```diff
@@ -159,3 +159,20 @@ proc masonTestHelp() {
   writeln("Tests pass if they exit with status code 0");
 }
 
+proc masonSystemHelp() {
+  writeln("Integrate a Mason package with system packages found via pkg-config");
+  writeln();
+  writeln("Usage:");
+  writeln('    mason system [options] [<args>...]');
+  writeln("    mason system [options]");
+  writeln();
+  writeln("Options:");
+  writeln("    pc                          Print a system packages .pc file");
```

Comment:
We'll want `--help` outputs for each subsubcommand here as well. `mason system pc --help` fails catastrophically, e.g.